spec/models/spotlight/solr_document/atomic_updates_spec.rb

```diff
@@ -15,6 +15,17 @@
       allow(subject).to receive_messages(to_solr: { id: 'doc_id', a: 1, b: 2 })
     end
 
+    context "when the index is not writable" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.